dev/dartdoc.dart

```diff
@@ -56,6 +56,22 @@ dependencies:
     args.add(name.substring(0, name.length - 5));
   }
 
+  [
```

Comment:
It would be great to future proof this. What about grabbing the list of sky_services libraries from `bin/cache/pkg/sky_services/lib/`? We should be able to infer the lib names based on the library file names.